src/Dfc.CourseDirectory.WebV2/Features/TLevels/ViewAndEditTLevel/ViewTLevel.cshtml

```diff
@@ -8,7 +8,7 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-three-quarters">
```

Comment:
I thought all of our other views were two-thirds?

Reply:
Corresponding page on Add T Level is also three quarters.